psh/test.yaml

```diff
@@ -1,7 +1,7 @@
 test:
     targets:
-      #TODO: remove when zynq7000-qemu target will stop being experimental
-      include: [armv7a9-zynq7000-qemu]
+      #TODO: remove when armv7a9-zynq7000-qemu and riscv64-generic-qemu targets will stop being experimental
+      include: [armv7a9-zynq7000-qemu, riscv64-generic-qemu]
```

Comment:
Did You run these tests for `riscv64-generic-qemu`? I am curious about results, if everything works fine I will enable it in CI.

Reply:
Yes, I did. All `psh` tests pass except for the `auth` test - that's the reason why we shouldn't enable `riscv64-generic-qemu` target quite yet.